acceptor_unix.go

```diff
@@ -36,6 +36,7 @@ func (svr *server) acceptNewConnection(fd int) error {
 		if err == unix.EAGAIN {
 			return nil
 		}
+		svr.opts.Logger.Warnf("Accept() fails due to error: %v", err)
```

Comment:
Sorry, but after a second thought, I think it's better to change the logging level to `Errorf`.